frontend/src/metabase/search/components/SearchSidebar/SearchSidebar.tsx

```diff
@@ -3,52 +3,58 @@ import _ from "underscore";
 import type {
   FilterTypeKeys,
   SearchFilterPropTypes,
-  SearchFilters,
+  URLSearchFilterQueryParams,
   SearchSidebarFilterComponent,
 } from "metabase/search/types";
-import { Title, Flex } from "metabase/ui";
+import { Button, Stack, Menu } from "metabase/ui";
 import { SearchFilterKeys } from "metabase/search/constants";
 import { SidebarFilter } from "metabase/search/components/SidebarFilter/SidebarFilter";
 import { TypeFilter } from "metabase/search/components/filters/TypeFilter/TypeFilter";
+import { CreatedByFilter } from "metabase/search/components/filters/CreatedByFilter/CreatedByFilter";
+
+type SearchSidebarProps = {
+  value: URLSearchFilterQueryParams;
```

Comment:
Changed the type here from `SearchFilters` since we always get strings back from the URL. We'll work with this in the `SearchSidebar` to handle these values through each filter